slackbot.go

```diff
@@ -0,0 +1,109 @@
+package main
+
+import (
+	"log"
+	"time"
+
+	pagerduty "github.com/PagerDuty/go-pagerduty"
+	"github.com/shomali11/slacker"
+)
+
+// Init TODO
+func Init(config *Config) {
+	log.Println("Connected!")
+	client := pagerduty.NewClient(config.PagerDuty.APIKey)
```

Comment:
I would make sure that client is only initialized once per program run, not per command execution.

Reply:
That's already happening. The Init function is only execute when the program is started.

Subsequent commands execution will only trigger the appropriate associated function.